code/datums/movement/mob.dm

```diff
@@ -25,9 +25,9 @@
 	mob.ghostize()
 
 // Incorporeal/Ghost movement
-/datum/movement_handler/mob/incorporeal/DoMove(var/direction)
+/datum/movement_handler/mob/incorporeal/DoMove(var/direction, var/mob/mover)
```

Comment:
forgot to pass mover, I suppose

Reply:
No, that's fine. But I am confused, why do wee need to AdjustMovementDirection for ghosts. They can be shocked or drunk :D?

Reply:
Doubledead